app/middleware/logging.py

```diff
@@ -53,7 +53,7 @@ def __call__(self, request):
         # HttpResponse and JSONResponse sure have
         # (e.g. WhiteNoiseFileResponse doesn't)
         if isinstance(response, (HttpResponse, JsonResponse)):
-            extra["response"]["payload"] = str(response.content)[:200]
+            extra["response"]["payload"] = response.content[:200].decode()
```

Comment:
out of interest: is there a difference between using `str()` and `.decode()`?

Reply:
@boecklic yes, see below

```python
my_byte_string = b'My string'

print(str(my_byte_string))
# "b'My string'"

print(my_byte_string.decode()
# "My string"
```